meta-luneos/recipes-webos-ose/sam/sam/sam.service

```diff
@@ -29,7 +29,3 @@ EnvironmentFile=-/var/systemd/system/env/sam.env
 ExecStartPre=/lib/systemd/system/scripts/sam.sh
 ExecStart=/usr/sbin/sam
 Restart=on-failure
-TimeoutStopSec=5
```

Comment:
this is a workaround you introduce to reduce the shutdown time

Reply:
We might not need it anymore, let's see how it behaves without.